sycl/test-e2e/lit.cfg.py

```diff
@@ -155,14 +155,18 @@
     config.available_features.add("gpu-intel-dg1")
 if lit_config.params.get("gpu-intel-dg2", False):
     config.available_features.add("gpu-intel-dg2")
-if lit_config.params.get("gpu-intel-pvc", False):
-    config.available_features.add("gpu-intel-pvc")
+if lit_config.params.get("gpu-intel-pvc", False) or \
+   lit_config.params.get("gpu-intel-pvc-vg", False) :
```

Comment:
i thought it would make it cleaner but yeah probably it did not, fixed in latest commit i think

i removed the multiline if so the second comment should be indirectly addressed

thx!